CONTRIBUTING.md

```diff
@@ -0,0 +1,250 @@
+
+If you adding new demo, please provide auto-testing support too:
+- add demo launch parameters in [demos/tests/cases.py](demos/tests/cases.py)
+- prepare list of input images in [demos/tests/image_sequences.py](demos/tests/image_sequences.py)
+
+*After this step you'll get **demo** for your model (if no demo was available)*
+
+## Accuracy validation
+
+Accuracy validation can be performed by [Accuracy Checker](./tools/accuracy_checker) tool, provided with repository. This tool can use IE to run converted model or original framework to run original model. Accuracy Checker supports lot of datasets, metrics and preprocessing options, what makes validation quite simple (if task is supported by tool). You need only create configuration file, which contain necessary parameters to do accuracy validation (specify dataset and annotation, pre- and post processing parameters, accuracy metric to compute and so on). More details you can find [here](./tools/accuracy_checker#resting-new-models).
+
+If model uses dataset which is unsupported by Accuracy Checker, you also must provide link to it. Please notice this issue in PR description. Don't forget about dataset license too (see [above](#how-to-contribute-model-to-open-model-zoo)).
+
+When the configuration file is ready, you must run Accuracy Checker to obtain metric results. If they match your results, that means  conversion was fully successful and Accuracy Checker fully supports your model, metric and dataset. If no - recheck [conversion](#model-conversion) parameters or validation configuration file.
+
+*After this step you will get accuracy validation configuration file - **<model_name>.yml***
+
+### Example
+
+In this [example](models/public/densenet-121-tf/model.yml) classification model DenseNet-121\*, pretrained in TensorFlow\*, is downloading from Google Drive\* as archive.
+
+```
+description: >-
+  This is an Tensorflow\* version of `densenet-121` model, one of the DenseNet
+  group of models designed to perform image classification. The weights were converted
+  from DenseNet-Keras Models. For details see repository <https://github.com/pudae/tensorflow-densenet/>,
+  paper <https://arxiv.org/pdf/1608.06993.pdf>
+task_type: classification
+files:
+  - name: tf-densenet121.tar.gz
+    size: 30597420
+    sha256: b31ec840358f1d20e1c6364d05ce463cb0bc0480042e663ad54547189501852d
+    source:
+      $type: google_drive
+      id: 0B_fUSpodN0t0eW1sVk1aeWREaDA
+postprocessing:
+  - $type: unpack_archive
+    format: gztar
+    file: tf-densenet121.tar.gz
+model_optimizer_args:
+  - --reverse_input_channels
+  - --input_shape=[1,224,224,3]
+  - --input=Placeholder
+  - --mean_values=Placeholder[123.68,116.78,103.94]
+  - --scale_values=Placeholder[58.8235294117647]
+  - --output=densenet121/predictions/Reshape_1
+  - --input_meta_graph=$dl_dir/tf-densenet121.ckpt.meta
+framework: tf
+license: https://raw.githubusercontent.com/pudae/tensorflow-densenet/master/LICENSE
+```
```

Comment:
I don't think we need a copy of the config here - the link works just fine.

Reply:
This example (and any other) is crucial for documentation. They will stay anyway.

Reply:
I'm not asking you to remove the example. I'm just asking that you don't duplicate it.